.eslintrc.json

```diff
@@ -0,0 +1,46 @@
+{
+  "parser": "@typescript-eslint/parser",
+  "plugins": ["@typescript-eslint"],
+  "extends": [
+    "eslint:recommended",
+    "plugin:react/recommended",
```

Comment:
I'd recommend considering:

`plugin:import/errors`, `plugin:react-hooks/recommended` and `plugin:jsx-a11y/recommended`

They've all improved my life at some point in the last couple of projects.

Reply:
I remember a slight run in with `jsx-a11y` settings in a previous project, but these are all good to include 🙂